### PR TITLE
Adding CLUU to the list of OSes

### PR DESCRIPTION
**Pull request by valibali**
This PR adds CLUU into the OS list in the Showcase chapter.

### CODE REVIEW

**Comment by valibali** [comment]
Thanks for the comments!

I did an update, I hope it's sufficient. If not, please comment.

File: content/showcase/cluu/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+title = "CLUU"
+date = 2023-06-14
+
+[extra]
+authors = ["valibali"]
++++
+
+# CLUU (Compact Lightweight Unix Utopia)
+
+[![License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE) [![Documentation](https://img.shields.io/badge/documentation-RUSTDOCS-blue.svg)](https://valibali.github.io/cluu/) [![Repository](https://img.shields.io/badge/repository-GitHub-green.svg)](https://github.com/your-username/cluu)
```
Comment (phil-opp) [inline content/showcase/cluu/index.md:11]: The license and repository links need to be fixed.

File: content/showcase/cluu/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+- **Kernel Implementation**: Further development of the microkernel's core functionality, including process management, interprocess communication, and resource management.
+- **Device Drivers**: Continued development of drivers for essential hardware components, including keyboard, storage devices, and other peripherals.
+- **Filesystem Support**: Introducing filesystem support to enable file I/O operations and provide a foundation for user-level processes and applications.
+
+The project remains open-source, and contributions from the community are highly encouraged. If you're interested in exploring the code, contributing enhancements, or reporting issues, please visit the [GitHub repository](https://github.com/valibali/cluu).
+
+## License
+
+CLUU is licensed under the MIT License. See [LICENSE](LICENSE) for more information.
```
Comment (phil-opp) [inline content/showcase/cluu/index.md:56]: The license link here is broken too. It should point to your repo.

**Review by phil-opp — COMMENTED** [review]
Looks very good now, thanks for the quick updates!

I found a few broken links, otherwise this seems ready to be merged from my side.

**Comment by valibali** [comment]
Fixed!

**Review by phil-opp — APPROVED** [review]
Looks good, thanks a lot! Sorry for the delay!

**Pull request merged**